### DISCUSSION TITLE
O Dev Junior precisa aprender a se vender?

### DISCUSSION THREAD

**Discussion by ItaloCanturil** [frontendbr/forum#2036] [Discussão]
Koe, suave com tudo?

Vi uma publicação no linkedin onde falava que se tinha muitas vagas para pleno/senior enquanto para devs junior era mais escasso, nos comentarios dessa publicação um cara falava que fez estudos e não é por que as vagas são poucas para nós (devs junior) e sim por que os devs não sabem se vender.

Será que isso influencia? Ou realmente as vagas são poucas e exigentes demais para uma primeira oportunidade na area?

**Comment by ninetails** [comment]
Concordo em partes.

Tente se colocar na visão de um recrutador: como você vai saber se o candidato é ótimo pra vaga ou não?

Quando não se é Junior, a galera tem muita coisa pra botar no currículo: experiências e projetos anteriores, quais empresas e de quais setores o profissional passou, quais tecnologias e problemas enfrentados...

Já pra um Junior, a bagagem dele é o que ele estudou até então e o que ele fez de projetos pessoais para aprendizado.

Pense, se o candidato não tiver diferencial, qual o critério que seria utilizado para a contratação dado que, sendo um recrutador, você quer otimizar o que o desenvolvedor pode fazer pela empresa _versus_ o que ele está cobrando ou pela quantia que eu posso tirar do orçamento da empresa para a contratação.

Em empresas sérias (ideais, utópicas, que não contratam junior pq não querem pagar por um mais especialista), geralmente sabem que se o candidato está se aplicando a uma vaga de Junior é porque ele está buscando experiência.

Mas o que se vê muito iniciante com medo de colocar projetos pessoais no GitHub ou no LinkedIn, e quando um recrutador pergunta se a pessoa fez algo, geralmente ela responde negativamente. E acho que nisso até que teve esse _boom_ de _bootcamps_, pq forçam os estudantes a fazer um projeto, subir no GitHub, publicar na web, pra ter uma visibilidade. Não é necessário pagar por um _bootcamp_ para que o profissional iniciante mostre o que sabe.

E daí volto ao assunto inicial: se este iniciante não explicar ou mostrar o que sabe ou aprendeu, não vai ter visibilidade no mercado e consequentemente não terá seu diferencial na hora da contratação.

---

Sobre a última pergunta:

> Ou realmente as vagas são poucas e exigentes demais para uma primeira oportunidade na area?

Tem muita vaga que não é listada. A empresa lista quando não encontrou ninguém internamente. O que seria encontrar internamente? **Indicação.**

Pense em aumentar seu networking para também alcançar mais recrutadores, seja em eventos de tecnologia e meetups, grupos de estudo, até mesmo fazendo cursos.

---

**TL;DR**
- Deixe seu LinkedIn trincando, pegue inspiração em perfis bem colocados no mercado.
- Amplie seu _networking_.
- Mostre o que você faz, seja no GitHub, seja no LinkedIn fazendo um vídeo da funcionalidade que você fez...
- Não minta. Se te perguntam e você não sabe algo, seja franco e procure estudar assim que possível para que na próxima entrevista você não seja pego desprevenido. Mas que acima de tudo mostre que você tá alí pra aprender ainda mais.
